lib/iris/tests/stock/netcdf.py

```diff
@@ -72,11 +74,13 @@ def _add_standard_data(nc_path, unlimited_dim_size=0):
             unlimited_dim_size if dim == unlimited_dim_names[0] else size
             for dim, size in zip(dims, shape)
         ]
-        data = np.zeros(shape, dtype=var.dtype)
+        data = np.ones(shape, dtype=var.dtype)  # Do not use zero
```

Comment:
@pp-mo `Do not use zero`... why not?

Add a smidge of context to help the reader? :+1:

Reply:
:+1: